modules/tinymce/src/plugins/paste/main/ts/core/SmartPaste.ts

```diff
@@ -33,7 +33,7 @@ const isAbsoluteUrl = (url: string) => {
 };
 
 const isImageUrl = (editor: Editor, url: string) => {
-  return isAbsoluteUrl(url) && Arr.exists(Settings.getAllowedImageFileTypes(editor), (type) => Strings.endsWith(url, `.${type}`));
+  return isAbsoluteUrl(url) && Arr.exists(Settings.getAllowedImageFileTypes(editor), (type) => Strings.endsWith(url.toLowerCase(), `.${type}`));
```

Comment:
We really want a proper case insensitive lookup here, so both `type` and `url` need to be converted to lowercase here.

- [ ] Ensure this is a proper case insensitive comparison